modules/sdk/src/main/scala/org/tessellation/sdk/domain/snapshot/peerSelect.scala

```diff
@@ -0,0 +1,11 @@
+package org.tessellation.sdk.domain.snapshot
+
+import org.tessellation.schema.peer.Peer
+
+object peerSelect {
+  trait PeerSelect[F[_]] {
+    def select: F[Peer]
+  }
```

Comment:
there is no need to additionally wrap the trait into an object

Reply:
The trait is in the object because I wanted to keep the logger name with it.

Reply:
You can Keep it in the same file but you can move trait out of object and name the file PeerSelect

Reply:
```scala
package org.tessellation.sdk.domain.snapshot

import org.tessellation.schema.peer.Peer

trait PeerSelect[F[_]] {
  def select: F[Peer]
}

object PeerSelect {
  val peerSelectLoggerName = "PeerSelectLogger"
}
```

Reply:
The trait has been moved out of the object and the object has been made a companion object.